<ui version="4.0" >
 <class>winLog</class>
 <widget class="QDialog" name="winLog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QTextBrowser" name="txtLog" >
   <property name="geometry" >
    <rect>
     <x>5</x>
     <y>5</y>
     <width>710</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave" >
   <property name="geometry" >
    <rect>
     <x>625</x>
     <y>361</y>
     <width>90</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>361</y>
     <width>90</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart" >
   <property name="geometry" >
    <rect>
     <x>435</x>
     <y>361</y>
     <width>90</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text" >
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
